integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.40 Version bump reason: nRF Connect SDK update
+0.5.41 Version bump reason: ESP-IDF update to v4.4-beta1 pre-release
```

Comment:
@andy31415 Please check if this is okay here or should be done in a follow-up PR

Reply:
This is ok and even required here (since we change the dockerfile, image should be updated).